lib/jazzy/themes/fullwidth/assets/js/jazzy.js

```diff
@@ -0,0 +1,37 @@
+window.jazzy = {'docset': false}
+if (typeof window.dash != 'undefined') {
+  document.documentElement.className += ' dash'
+  window.jazzy.docset = true
+}
+if (navigator.userAgent.match(/xcode/i)) {
+  document.documentElement.className += ' xcode'
+  window.jazzy.docset = true
+}
+
+// On doc load, toggle the URL hash discussion if present
```

Comment:
Not quite. All that serves the purpose of toggling the tokens when clicked instead of refreshing the page, and only in the browser and all tokens are displayed by default in dash.